src/any_agent/tools/wrappers.py

```diff
@@ -100,8 +105,33 @@ async def wrap_mcp_server(
 }
 
 
-async def import_and_wrap_tools(
-    tools: list[str | dict], agent_framework: AgentFramework
+def verify_callable(tool: Callable) -> None:
```

Comment:
do we need to do this ourselves? Doesn't each framework already do this check (and might have different constraints)?

Reply:
Smolagents does this check immediately, but other tools like llama index didn't throw an error right away if they were missing. So, I went with adding this ourselves so that we can give immediate feedback about the issue. I didn't make it exhaustive (e.g. I don't check that the docstring has correct content like a description of each arg, I don't check that the docstring is in the google docstring format, etc).